docs/home/release-notes/v1.50-v1.59.md

```diff
@@ -3,7 +3,13 @@
 ## Version 1.59
 
 !!! tip ""
-    Helm chart version 1.59.x
+    Helm chart version 1.59.0
+
+### Helm configuration changes
+
+The following ODM components were removed: `clickhouse`, `mysql-prepare` and `clickhouse-helper`.
+
+Removing the old `clickHouse` instance is completely safe, as its data was migrated to the cluster version of ClickHouse in the 1.58 release, and it should have been disabled after that.
```

Comment:
[Optional] we can mention new clickhouse container names here, so an admin will see that we are not getting rid of CH

Reply:
We can, but I’m not sure it makes sense.

In version 1.58, a forced migration to the ClickHouse cluster occurs, along with switching ODM to it. Therefore, the old ClickHouse (even if it has been running all this time) contains an outdated set of data, making its removal harmless.

As for the new ClickHouse cluster, it cannot be accidentally deleted since it requires a deliberate action to disable its use in the settings.